Documentation/Specification/silica.tex

```diff
@@ -1249,14 +1249,14 @@
 This macro will signal a runtime error if sheet is not a subclass of
 \cl{sheet-with-medium-mixin}.
 
-The \arg{medium} argument is not evaluated, and must be a symbol that is bound
+The \arg{medium} argument is not evaluated, and must be a symbol that will be bound
 to a medium.  \arg{body} may have zero or more declarations as its first forms.
```

Comment:
I had doubts about that, so checked how it's implemented. All looks fine though, `will be` describes the behavior better (also, concurring interpretation would cease reason of `with-sheet-medium-bound` existance).